src/groq/index.ts

```diff
@@ -111,4 +119,26 @@ export default class GroqChat {
       }
     }
   }
+
+  /**
+   * Logs the token usage information to the console. by args --token-usage or -tu
+   *
+   * @param tokenUsageInfo - The token usage information object.
+   */
+  public saveTokenUsageInfo(tokenUsageInfo: Groq.CompletionUsage): void {
+    console.log('Token Usage Information:\n');
+
+    // Available properties:
+    //console.error(`queue_time: ${tokenUsageInfo.queue_time}`);
+    //console.error(`usage_prompt_tokens: ${tokenUsageInfo.prompt_tokens}`);
+    //console.error(`prompt_time: ${tokenUsageInfo.prompt_time}`);
+    //console.error(`completion_tokens: ${tokenUsageInfo.completion_tokens}`);
+    //console.error(`completion_time: ${tokenUsageInfo.completion_time}`);
+    //console.error(`total_time: ${tokenUsageInfo.total_time}`);
+    //console.error(`total_tokens: ${tokenUsageInfo.total_tokens}`);
+
+    console.error(`Completion Tokens: ${tokenUsageInfo.completion_tokens}`);
+    console.error(`Usage Prompt Tokens: ${tokenUsageInfo.prompt_tokens}`);
```

Comment:
issue: Right now, these display above the output of every file processed, i.e.:

![image](https://github.com/user-attachments/assets/5e40a95e-82ce-4df9-9d40-0970ec630d82)

When `optimizeit` processes several files at once the output can be very big. This would then force the user to scroll all the way up to see the tokens which is not user friendly. I think that the token info needs to be displayed at the `bottom most` of the program output, i.e.:

![image](https://github.com/user-attachments/assets/9e2c98f7-e830-4077-a3b4-9687bbfa80c5)